enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -95,263 +112,214 @@ This is an example of an OKD cluster hosting solution where several organization
 
 #### As an OKD admin, I need to forward my audit logs to a secure SIEM that meets government regulations
 
-This is often required for industries such as the US public sector, healthcare or financials. The logs will be forwarded to a government approved SIEM through secure means (mutual TLS).
+This is often required for industries such as the US public sector, healthcare or financials.: The logs will be forwarded to a government approved SIEM through secure means (mutual TLS).
 
 ### Implementation Details
 
-#### Assumptions
-
-* A pipeline shall support at least one output.
-* A pipeline may support multiple outputs.
-* Outputs that require secrets which are not created by the cluster-logging-operator shall be created and managed by the administrator of the endpoint
-* Secrets shall have keys of: `tls.crt`, `tls.key`, `ca-bundler.crt` which point to the respective certificates for which they represent. Secrets shall have the key `shared_key` for use when using `forward` in a secure manner
-* The operator will only support a single instance of log forwarding named `instance` for at least the tech preview release
-* Log forwarding is enabled for tech preview when the instance of `clusterloggging` is annotated with: `clusterlogging.openshift.io/logforwardingtechpreview: enabled`
+* A pipeline associates multiple input names with multiple output names.
+* Users cannot define new inputs (for initial release)
+* Users can define new outputs, output configuration includes:
+  - `type` (e.g. `Syslog`, `Fluent`)
+  - `destination` URL for the endpoint.
+  - `secretRef` referring to a secret object used for secure connections.
+    Secrets not created by the cluster-logging-operator shall be created and managed by the administrator of the endpoint
```

Comment:
Should we be explicit the secret needs/should live in the CLO namespace.  We will otherwise need to grant broader permissions to the CLO

Reply:
Yes lets. I'll also make it clear that if an output.secretRef has no namespace it defaults to the CLO namespace. We can decide how/if we use secrets in other namespaces post GA.

Reply:
+1 to that